src/main/java/org/synyx/urlaubsverwaltung/security/WebMvcConfig.java

```diff
@@ -37,23 +37,6 @@ public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(localeInterceptor);
     }
 
-    @Override
-    public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry
-            .addResourceHandler("/assets/**", "/css/**", "/images/**")
-            .addResourceLocations("classpath:static/assets/", "classpath:static/css/", "classpath:static/images/")
-            .setCacheControl(CacheControl.maxAge(365, TimeUnit.DAYS))
-            .resourceChain(false)
-            // do not add contentVersionStrategy for "/assets/**" since these files are already handled by the webpack build
-            .addResolver(new VersionResourceResolver().addContentVersionStrategy("/css/**", "/images/**"))
```

Comment:
Gibt es Dinge im `/images/**` Verzeichnis die man ebenfalls in den `spring.resources.chain.strategy.content.paths` aufnehmen will?